app/components/MainScreen/index.js

```diff
@@ -166,9 +209,14 @@ const MainScreen = ({ navigation }) => {
                 isUserLogin={isUserLogin}
                 testID={'btn_login'}
                 accessibilityLabel={'Login/Sign Up'}
-                photo={userPhoto}
+                photo={
+                  cdnUrls && cdnUrls.cache && userInfo.image
+                    ? `${cdnUrls.cache}/profile/avatar/${userInfo.image}`
```

Comment:
Be aware that despite my fix this does not work yet reliable for the development backend as the url given by the config API call are not ok for the development backend - for the _planet-webapp_ we directly configured the CDN URL within the .env at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/d4928069530a5ee401b03c3958af29b6e53d78f6/.env.local.example#L6 and used that for building the image path at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/utils/getImageURL.ts - I would prefer keeping the config API call as on production this should work correctly.